core/controller/src/main/scala/org/apache/openwhisk/core/controller/AuthorizedRouteDispatcher.scala

```diff
@@ -86,7 +89,11 @@ trait BasicAuthorizedRouteProvider extends Directives {
               case NotFound =>
                 handleEntitlementFailure(
                   RejectRequest(NotFound, Some(ErrorResponse(Messages.resourceDoesntExist(resource.fqname), transid))))
-              case _ => handleEntitlementFailure(t)
+              case code =>
+                if (code == TooManyRequests) {
+                  logger.warn(this, s"${resource.id} invoked failed: ${t}")
+                }
```

Comment:
Since  we don't have a logger at hand here:

1. Should we pass a logger into the `authorizeAndDispatch` method?
2. Can we log outside of the method by inspecting it's return value?

Reply:
Just add `protected implicit val logging: Logging` in `trait BasicAuthorizedRouteProvider` is enough,
because the subclass will implement the logging.

Reply:
There was... it's now a debug level statement. @ningyougang does this not suffice? (we can refactor the debug statement so that the concurrent throttle check emits a more specific messages.)

https://github.com/apache/incubator-openwhisk/blob/be1e3a19c02956c9be85023a0bb0ff399c21444d/core/controller/src/main/scala/org/apache/openwhisk/core/entitlement/ActivationThrottler.scala#L41-L50

Reply:
I see. It’s possible to thread the resource through to the rate throttler which would have the name of the action if it’s important.